eclipse-jdt/eclipse.jdt.ui
# [WIP] - New proposal split branch

## What it does

Takes the reverted split of proposals patches from UI to Core and builds upon those patches to solve the remaining issues.
See #835

## How to test

See issue and attempt to actually apply proposals/quick fixes and verify no errors occur.

## Author checklist
